core/src/main/kotlin/org/evomaster/core/problem/graphQl/Data.kt

```diff
@@ -0,0 +1,11 @@
+class Data {
+
+    var __schema: __Schema? = null
+
+
+    override fun toString(): String {
+            return "data: {  ${this.__schema}}"
```

Comment:
you can mark those classed with the Kotlin keyword `data`, which then will automatically handle proper `toString` (so you do not need to handle them manually)

Reply:
done